x-pack/plugins/observability_solution/observability/public/components/custom_threshold/components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -164,7 +194,21 @@ export default function AlertDetailsAppSection({
             defaultMessage: 'Source',
           }
         ),
-        value: <Groups groups={groups} />,
+        value: (
+          <>
+            <Groups groups={groups} timeRange={timeRange} />
+            <span>
+              <a href={viewInAppUrl} target="_blank">
```

Comment:
You can use `EuiLink` here instead.

Reply:
Thanks for the suggestion, updated in https://github.com/elastic/kibana/commit/1460a3182799a95a2d0a62f9ae07e06375dd4da7

Looks better with icon next to link indicating it opens in a new tab.

<img width="1043" alt="Screenshot 2024-03-14 at 22 57 33" src="https://github.com/elastic/kibana/assets/69037875/8dde8f25-5fe2-479a-bba5-61dd46e3022b">